docs/configuration/radio/power.mdx

```diff
@@ -19,7 +19,13 @@ The power config options are: Power Saving, Shutdown after losing power, ADC Mul
 
 ### Power Saving
 
-If set, Bluetooth, Wifi, and screen (if applicable) are turned off. When using the Router device role, this setting is on by default. The assumption for a Router is that the device will be used in a standalone manner where these features are not needed, and will be shut off to conserve power. This is especially useful when a device is powered from a low-current source (i.e. solar).
+:::warning 
+Enabling Power Saving will disable Bluetooth, Serial, and WiFi, preventing further changes to the device. Press the user button or reset the device to make changes.
```

Comment:
I definitely like your admonition more, I had real trouble figuring this out.

I found the existing description less than ideal though, so I would like to change it. I'm not tied to what I have in there now, just felt it could be improved.

Reply:
Love it. ♻️